app/build.gradle

```diff
@@ -62,6 +62,7 @@ dependencies {
       'androidx.constraintlayout:constraintlayout:1.1.3',
       'androidx.core:core-ktx:1.0.2',
       'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-alpha03',
+      'com.google.android.material:material:1.0.0',
```

Comment:
Maintain the alphabetical order while adding libraries

Reply:
Resolved